bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -117,6 +118,7 @@ public Response doCreate( final String packageType, final String type, final Str
                 builderModifier.accept( builder );
             }
             response = builder.build();
+            contentController.generateHttpMetadataHeaders( transfer, request, response );
         }
```

Comment:
@jdcasey For this, I put the metadata generation when storing the content (now it effects all the packagetypes), not for npm hosted content GET request as we discussed in last meeting.

That's because I address the issue why a http-metadata.json generated for npm publish not works. I said before that maybe npm doesn't recognize or admit the headers, after futher trials, it's confirmed to be caused by the LAST-MODIFIED field not states there correctly in the response headers. so I force it here using ResponseBuiler.lastModified( new Date( target.lastModified() ) ), and it works.

Another reason I put the HttpExchangeMetadata generation when storing is, PUT requests may be not as much as GET, saving the resources of metadata generation for GETs should be a sensible choice here...